### PR TITLE
fix(delivery): Add more context for dispatch notification email + formatting fixes

### PR DESCRIPTION
**Pull request by Alchez**
**Ref:** https://github.com/DigiThinkIT/bloomstack/issues/178

### CODE REVIEW

File: erpnext/stock/doctype/delivery_trip/delivery_trip.py [hunk 1]
```diff
@@ -392,15 +374,19 @@ def notify_customers(delivery_trip):
...
 
 	email_recipients = []
 
 	for stop in delivery_trip.delivery_stops:
-		contact_info = frappe.db.get_value("Contact", stop.contact,
-											["first_name", "last_name", "email_id", "gender"], as_dict=1)
+		contact_info = frappe.db.get_value("Contact", stop.contact, ["first_name", "last_name", "email_id"], as_dict=1)
+
+		if stop.sales_invoice:
+			items = frappe.get_all("Sales Invoice Item", filters={"parent": stop.sales_invoice, "docstatus": 1}, fields=["*"])
```
Comment (vjFaLk) [inline erpnext/stock/doctype/delivery_trip/delivery_trip.py]: Get Items from Delivery Note, not Sales Invoice

File: erpnext/stock/doctype/delivery_trip/delivery_trip.py [hunk 2]
```diff
@@ -392,15 +374,19 @@ def notify_customers(delivery_trip):
 	delivery_trip = frappe.get_doc("Delivery Trip", delivery_trip)
 
 	context = delivery_trip.as_dict()
+	context.pop("delivery_stops", None)
```
Comment (vjFaLk) [inline erpnext/stock/doctype/delivery_trip/delivery_trip.py]: Remove this line

**Review by vjFaLk — CHANGES_REQUESTED** [review]

**Comment by vjFaLk** [comment]
Upstream?

**Comment by vjFaLk** [comment]
@Alchez PLS UPSTREAM

**Comment by Alchez** [comment]
Made upstream to [hotfix](https://github.com/frappe/erpnext/pull/17966) and [develop](https://github.com/frappe/erpnext/pull/17967).

**Review by vjFaLk — APPROVED** [review]

**Pull request merged**